### PR TITLE
Add one vcr test

### PR DESCRIPTION
**Pull request by maelle**
:wave: @steffilazerte! Here's my shameless vcr promotion effort! :wink:

I ran `vcr::use_vcr()` and then created an example test based off the first part of an existing test.

### CODE REVIEW

File: tests/fixtures/weather_html_hour.yml [hunk 1]
```diff
@@ -0,0 +1,854 @@
+http_interactions:
```
Comment (maelle) [inline tests/fixtures/weather_html_hour.yml]: In this file the http request and response is saved. A cassette could contain several several requests and responses if the code inside "use_cassette()" made several requests.

Comment (maelle) [reply tests/fixtures/weather_html_hour.yml]: Note that the diff of cassettes can be hidden if one edits the `.gitattributes` of the repo.

```
* text=auto

tests/fixtures/**/* -diff
```

File: tests/testthat/test_04b_weather_raw.R [hunk 2]
```diff
@@ -0,0 +1,20 @@
+context("weather_raw with vcr")
```
Comment (maelle) [inline tests/testthat/test_04b_weather_raw.R]: I ran the test once to create the cassette, then I turned off wifi and re-ran the test to convince myself it hard worked.

**Comment by steffilazerte** [comment]
Thanks! I'm going to leave this PR here as a reference and will hopefully have some time to take a look in the near future. If I get things figured out, perhaps for that blog post, I could talk a bit about using both/either `vcr` and `mockery` to solve my internet connection issues!

**Comment by maelle** [comment]
> perhaps for that blog post, I could talk a bit about using both/either vcr and mockery to solve my internet connection issues!

No pressure :angel: but that sounds like an awesome blog post topic :heart_eyes_cat:

**Comment by maelle** [comment]
One last thing that might be useful to you since you want to check real API things regularly ([which I learnt in your comm call talk](https://twitter.com/ma_salmon/status/1202653885107712003) :wink:): https://docs.ropensci.org/vcr/reference/lightswitch.html (see e.g. [a config file where vcr is turned off for one build](https://github.com/lockedata/hubspot/blob/master/.travis.yml#L33)).

**Comment by steffilazerte** [comment]
Oh that's a great option! Especially being able to integrate it into Travis CI and AppVeyor!

**Comment by steffilazerte** [comment]
I switched all the tests to vcr and CRAN had no problem. I did have to think hard sometimes, to make sure that the test fixtures remained small. But everything worked out well!

**Comment by maelle** [comment]
This is so cool!!

Do you mean making the test cases small or editing the fixtures by hand?

**Comment by steffilazerte** [comment]
Definitely not editing by hand! I meant re-using examples for different tests and strategically picking dates (i.e. daily  data in ECCC is downloaded by month, so instead of a date range of 2018-09-01 to 2018-10-01, which downloads two months, I would change that to 2018-09-31, which downloads one mont, unless the point was to test downloading multiple months.)

**Comment by maelle** [comment]
thank you! :pray:

**Pull request closed without merging**